<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_9" stretch="1,5,1,1">
      <item>
       <widget class="QTabWidget" name="tabWidgetFunction">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabGrayscale">
         <attribute name="title">
          <string>灰度</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="cboxCvtToGrayScale">
              <property name="text">
               <string>转化为灰度图</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>描边</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QCheckBox" name="checkBox_9">
              <property name="text">
               <string>勾画边缘（这会把图片转化为灰度图）</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_14">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabResize">
         <attribute name="title">
          <string>调整尺寸</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5,0">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>宽 Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>高 Height</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_13" stretch="0,1,0,1">
            <item>
             <widget class="QSlider" name="sliderW">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSliderWInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>w</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="sliderH">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSliderHInfo">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>h</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QGroupBox" name="groupFixedRatio">
            <property name="title">
             <string>选项</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="0,2,3">
             <item>
              <widget class="QCheckBox" name="cboxFixedRatio">
               <property name="text">
                <string>固定缩放比</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_14">
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>像素</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">bGroupRatioOrNot</string>
                 </attribute>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_2">
                 <property name="text">
                  <string>百分比</string>
                 </property>
                 <attribute name="buttonGroup">
                  <string notr="true">bGroupRatioOrNot</string>
                 </attribute>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>自定义</string>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <widget class="QSpinBox" name="sBoxResizeW"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelResizeWUnit">
                   <property name="text">
                    <string>Pixel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6">
                 <item>
                  <widget class="QSpinBox" name="sBoxResizeH"/>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelResizeHUnit">
                   <property name="text">
                    <string>Pixel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>透明色</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QCheckBox" name="checkBox_4">
                <property name="text">
                 <string>深色</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_5">
                <property name="text">
                 <string>浅色</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QSlider" name="horizontalSlider_13">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>高级</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_2">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>R</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>G</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>B</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_6">
                <property name="text">
                 <string>高于</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QSlider" name="horizontalSlider_15">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_16">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_17">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_7">
                <property name="text">
                 <string>低于</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QSlider" name="horizontalSlider_18">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_19">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSlider" name="horizontalSlider_20">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabImgInfo">
         <attribute name="title">
          <string>信息</string>
         </attribute>
         <widget class="QLabel" name="labelQueueNum">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>391</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>操作队列</string>
          </property>
         </widget>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShowImg">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>1920</width>
          <height>1080</height>
         </size>
        </property>
        <property name="text">
         <string>图片显示在这里</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelShowImgInfo">
        <property name="text">
         <string>Size:   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveImg">
        <property name="text">
         <string>保存</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>588</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileSetting">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="fileOpenAction"/>
    <addaction name="saveAsAction"/>
    <addaction name="recentFileAction"/>
    <addaction name="cleanHistoryAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>编辑</string>
    </property>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="fileSetting"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="convertToGrayScaleAction"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="action_6"/>
  </widget>
  <action name="fileOpenAction">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="recentFileAction">
   <property name="text">
    <string>最近打开</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>回退上一步</string>
   </property>
  </action>
  <action name="convertToGrayScaleAction">
   <property name="text">
    <string>线条</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>刻录时间轴</string>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>矩形</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>取色</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>文本</string>
   </property>
  </action>
  <action name="cleanHistoryAction">
   <property name="text">
    <string>清除历史打开记录</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bGroupRatioOrNot"/>
 </buttongroups>
</ui>
